### DISCUSSION TITLE
Support for Chinese

### DISCUSSION THREAD

**Discussion by Ghost-Cavendish** [zsviczian/obsidian-excalidraw-plugin#169] [Ideas]
Can you add support for Chinese? I see that the web version of Excalidraw supports Chinese. If you don’t have time, can you give me some suggestions? Thank you!

**Comment by zsviczian** [comment]
Hi @jk110, what do you mean by "support Chinese"? 
- are you looking for handwritten fonts: #14 
- or the menus in Chinese?
- or?

**Reply by Ghost-Cavendish** [reply]
Chinese menu and Chinese handwriting
